tests/test_create_merge_ids.py

```diff
@@ -43,24 +43,6 @@ def test_get_combined_map5(self):
         actual_result = sorted(create_match_sets(match_dir), key=lambda k: len(k))
         self.assertEqual(actual_result, expected_result)
 
-    def test_get_match_sets_with_extra_id(self):
```

Comment:
@jmelot write up why you got rid of this for Brian

Reply:
Ah ok I remember now. This test was meant to exercise the code that filters out matches that contain ids we no longer see in our current set of original ids. That logic existed because we wanted to make sure we weren't introducing any obsolete ids from the simhash index into the set of current matches. Because we're removing simhash, that logic and this test are not necessary.